$OpenBSD: patch-GNUmakefile_in,v 1.2 2007/06/06 20:36:23 kili Exp $

# No rsync, just use pax(1).
# This isn't of any use yet, but hopefully it will be in the future when
# we're able to build the documentation by ourselves.

--- GNUmakefile.in.orig	Sun Dec 24 17:45:50 2006
+++ GNUmakefile.in	Wed Dec 27 17:30:35 2006
@@ -57,7 +57,8 @@ doc: 
 
 install-WWW:
 	-$(INSTALL) -m 755 -d $(DESTDIR)$(webdir)
-	rsync -rl $(outdir)/offline-root/ $(DESTDIR)$(webdir)
+	pax -rw -s @^$(outdir)/offline-root@@ \
+		$(outdir)/offline-root $(DESTDIR)$(webdir)
 	$(MAKE) -C Documentation/user local-install-WWW
 	$(MAKE) -C Documentation/user install-info
 
